src/views/NoticeOfWithdrawal.vue

```diff
@@ -0,0 +1,1008 @@
+                </p>
+              </header>
+              <div
+                id="document-delivery-section"
+                :class="{ 'invalid-section': !documentDeliveryValid && showErrors }"
+              >
+                <v-card
+                  flat
+                  class="py-8 px-5"
+                >
+                  <DocumentDelivery
+                    editableCompletingParty="true"
+                    :contactValue="getBusinessEmail"
+                    contactLabel="Business Office"
+                    :documentOptionalEmail="documentOptionalEmail"
+                    @update:optionalEmail="documentOptionalEmail=$event"
+                    @valid="documentDeliveryValid=$event"
+                  />
+                </v-card>
+              </div>
+            </section>
+
+            <!-- Certify -->
+            <section>
+              <header>
+                <h2>Certify</h2>
+                <p class="grey-text">
+                  Confirm the legal name of the person authorized to complete and submit this withdrawal.
+                </p>
+              </header>
+              <div
+                id="certify-form-section"
+                :class="{ 'invalid-section': !certifyFormValid && showErrors }"
+              >
+                <Certify
+                  ref="certifyRef"
+                  :isCertified.sync="isCertified"
+                  :certifiedBy.sync="certifiedBy"
+                  :class="{ 'invalid-certify': !certifyFormValid && showErrors }"
+                  :disableEdit="!isRoleStaff"
+                  :entityDisplay="displayName()"
+                  :message="certifyText(FilingCodes.NOTICE_OF_WITHDRAWAL)"
+                  @valid="certifyFormValid=$event"
+                />
+              </div>
+            </section>
+
+            <!-- Withdrawal of Arrangement Records -->
+            <section>
+              <header>
+                <h2>Withdrawal of Arrangement Records</h2>
+                <p class="grey-text">
+                  Indicate if the record to be withdrawn is pursuant to a plan of arrangement and
+                  if one or more of the provisions of the arrangement have come into effect.
+                </p>
+              </header>
+              <div
+                id="court-order-section"
+                :class="{ 'invalid-section': showErrors }"
+              >
+                <v-card
+                  flat
+                  class="py-8 px-5"
+                >
+                  <PlanOfArrangement
+                    :autoValidation="showErrors"
+                    :hasDraftPlanOfArrangement="hasPlanOfArrangement"
+                    :hasDraftComeIntoEffect="hasComeIntoEffect"
+                    @emitPoa="hasPlanOfArrangement=$event"
+                    @emitEffect="hasComeIntoEffect=$event"
+                  />
+                </v-card>
```

Comment:
I saw Jacqueline removed this section from design last Friday, will need to confirm with her tmr
![image](https://github.com/user-attachments/assets/c16c4511-ee01-4acc-805f-959d02c058bb)